packages/flutter_tools/templates/app/analysis_options.yaml.tmpl

```diff
@@ -0,0 +1,30 @@
+# This file configures the analyzer, which statically analyzes Dart code to
+# check for errors, warnings, and lints.
+#
+# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
+# IDEs (https://dart.dev/tools#ides-and-editors). The analyzer can also be
+# invoked from the command line by running `flutter analyze`.
+
+# The following line activates a set of recommended lints for Flutter apps
+# and packages designed to encourage good coding practises.
+include: package:lints/flutter.yaml
+
+# TODO(goderbauer): Uncomment "linter:" and "rules:" when https://github.com/dart-lang/linter/issues/2612 is fixed.
```

Comment:
if we're landing before this gets fixed, let's put the TODO (and maybe the commented-out code?) somewhere where it won't get stuck in people's analysis options for years.

Reply:
I will fix this before submitting it. The Fix has already landed in the Dart SDK and engine, just waiting for a roll into the framework.